src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpSyntaxFormattingService.cs

```diff
@@ -19,37 +19,24 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Formatting
 {
-#if !CODE_STYLE
-    [ExportLanguageService(typeof(ISyntaxFormattingService), LanguageNames.CSharp), Shared]
-#endif
-    internal class CSharpSyntaxFormattingService : AbstractSyntaxFormattingService
+    internal class CSharpSyntaxFormatting : AbstractSyntaxFormatting
```

Comment:
Rename the file to `CSharpSyntaxFormatting.cs` and also the VB one to `VisualBasicSyntaxFormatting.vb`

Reply:
yes.  that makes sense.  will do in followup!

Reply:
done in  https://github.com/dotnet/roslyn/pull/60462